jobs/build/ocp/Jenkinsfile

```diff
@@ -595,134 +595,136 @@ node(TARGET_NODE) {
                 }
             }
 
-            stage("origin-web-console repo") {
-                sh "go get github.com/jteeuwen/go-bindata"
-                // defines:
-                //   WEB_CONSOLE_DIR
-                //   GITHUB_URLS["origin-web-console"]
-                //   GITHUB_BASE_PATHS["origin-web-console"]
-                buildlib.initialize_origin_web_console()
-                dir(WEB_CONSOLE_DIR) {
-                    // Enable fake merge driver used in our .gitattributes
-                    sh "git config merge.ours.driver true"
-                    // Use fake merge driver on specific directories
-                    // We will be re-generating the dist directory, so ignore it for the merge
-                    sh "echo 'dist/** merge=ours' >> .gitattributes"
+            if (BUILD_VERSION_MAJOR == 3) {
```

Comment:
I'd put this inside the `stage`. If you execute different stages on different runs, Jenkins does not display the pipeline consistently.